src/main/java/core/basesyntax/Main.java

```diff
@@ -2,7 +2,7 @@
 
 public class Main {
     private static FigureSupplier figureSupplier = new FigureSupplier();
-    private static Figure[] figures = new Figure[6];
+    private static final Figure[] figures = new Figure[6];
```

Comment:
not the whole array should be const, just make 6 a const variable and you can create this array inside the main method

Reply:
understood, and in general the array can be final, whether in this case it is bad?

Reply:
yes, an array can be final if needed. it all depends on your goal and how you want to use it

Reply:
Ok thanks